doc/source/dev/roadmap-detailed.rst

```diff
@@ -78,8 +78,8 @@ Pythran is still an optional build dependency, and can be disabled with
 happen it must be clear that the maintenance burden is low enough.
 
 
-Use of venerable Fortran libraries
```

Comment:
The trigram "venerable Fortran library" stands out to me in SciPy documentation. I thought "venerable" might not be needed every time we mention Fortran libraries.